webpack/components/ReportJsonViewer.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import JSONTree from 'react-json-tree'
+
+const theme = {
+  scheme: 'foreman',
+  backgroundColor: 'rgba(0, 0, 0, 0)',
+  base00: 'rgba(0, 0, 0, 0)',
+};
+
+class ReportJsonViewer extends React.Component {
+  render () {
+    console.log(this);
+    return <JSONTree data={this.props.data} hideRoot theme={theme} />
```

Comment:
Is this hiding the root on the module_args? I see the following 

![screenshot from 2018-02-26 09-02-57](https://user-images.githubusercontent.com/598891/36659278-36d60998-1ad4-11e8-8a4d-26bbb5643796.png)